src/main/java/Duke.java

```diff
@@ -1,10 +1,297 @@
+    }
+
+    private static void markJobAsDone(Task task) {
+        task.setDone(true);
+        System.out.print("Congrats! You've completed: \n   ");
+        task.printTask();
+        System.out.println();
+    }
+
+
+
+    /**
+     * PRINTING METHODS
+     * */
+
+    private static void printTaskAdded(Task task) {
+        System.out.println("Added to list: ");
+        task.printTask();
+        printNumTasksLeft();
+        System.out.println();
+    }
+
+    private static void printNumTasksLeft(){
+        String output = Integer.toString(Task.taskCount);
+        output += (Task.taskCount == 1)? " task" : " tasks";
+        output += " in the list";
+
+        System.out.println(output);
+    }
+
+    private static void printInvalidInputWarning() {
+        System.out.println("Wrong format! Enter \"help\" for a list of available commands and format\n");
+    }
+
+    private static void printNoTaskWarning() {
+        System.out.println("You don't have any tasks yet! Enter a task");
+        System.out.println("Enter \"help\" for a list of available commands and format\n");
+    }
+
+    private static void printInvalidTaskWarning(int jobNumber) {
+        String smaller = "Enter a valid job number. Use the list command to view your current tasks.";
+        String larger = "You don't have that many jobs! Use the list command to view your current tasks.";
+        System.out.println(jobNumber < 0 ? smaller : larger);
+        System.out.println("Enter \"help\" for a list of available commands and format\n");
+    }
+
+    private static void printHelp(){
+        String commandList = "LIST - \n" +
+                "FORMAT: list";
+
+        String commandDone = "DONE - \n" +
+                "FORMAT: done [(int) number]";
+
+        String commandTodo = "TODO - \n" +
+                "FORMAT: todo [(str) job]";
+
+        String commandDeadline = "DEADLINE - \n" +
+                "FORMAT: deadline [(str) job] /by [(str) deadline]";
+        String commandAdd = "ADD - \n" +
+                "FORMAT: [(str) job]";
+
+
+        System.out.println("COMMAND LIST:");
+        System.out.println("-------------");
+        System.out.println(commandAdd + '\n');
+        System.out.println(commandTodo + '\n');
+        System.out.println(commandDeadline + '\n');
+        System.out.println(commandList + '\n');
+        System.out.println(commandDone + '\n');
+        System.out.println("To exit, enter \"bye\"\n");
+
+    }
+
+    private static void printHello() {
+        // Start - Greets user
+        String line = "____________________________________________________________\n";
+        String hello_message = "Hello I'm Diuk! \nWhat would you like to do today?\n";
+
+        System.out.print(line);
+        System.out.print(hello_message);
+        System.out.print(line);
+    }
+
+    private static void printBye() {
+        String line = "____________________________________________________________\n";
```

Comment:
If you frequently use the line variable (e.g. in printHello and printBye functions), you should declare it as a constant.

Reply:
Good point, I'll consider this change!